sound/playwav.h

```diff
@@ -371,12 +389,22 @@ class PlayWav : StateMachine, public AudioStream {
     return filename_;
   }
 
+  const char* effectname() const {
+    return effectname_;
+  }
+  
+  const char* effect2ndname() const {
+     return effect2ndname_;
+  }  
+  
 private:
   volatile bool run_ = false;
   Effect* volatile effect_ = nullptr;
   Effect::FileID new_file_id_;
   Effect::FileID old_file_id_;
   char filename_[128];
+  const char* effectname_;
```

Comment:
isn't this the same as current_->name_ ?

Reply:
Yes, that should be the case